### PR TITLE
Use bot core paste service

### PR DESCRIPTION
**Pull request by ChrisLovering**
This is on top of https://github.com/python-discord/bot/pull/2642, so that will need reviewing, and merging first.

### CODE REVIEW

File: tests/bot/utils/test_services.py
Comment (mbaruh) [inline tests/bot/utils/test_services.py:1]: Should this be moved to botcore? I see we do have a test suite there.

Comment (ChrisLovering) [reply tests/bot/utils/test_services.py:1]: Yea, would likely make sense to live in there

**Review by mbaruh — APPROVED** [review]
Other than moving the tests, seems good to go

File: bot/resources/tags/paste.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ embed:
 If your code is too long to fit in a codeblock in Discord, you can paste your code here:
 https://paste.pythondiscord.com/
 
-After pasting your code, **save** it by clicking the floppy disk icon in the top right, or by typing `ctrl + S`. After doing that, the URL should **change**. Copy the URL and post it here so others can see it.
+After pasting your code, **paste** it by clicking the Paste! button in the bottom left, or by pressing `CTRL + S`. After doing that, you will be navigated to the new paste's page. Copy the URL and post it here so others can see it.
```
[the remark this answers is not in the document]
Comment (ChrisLovering) [reply bot/resources/tags/paste.md]: Yea, pointing it out, the sentence seems repetitive, I've applied that change in https://github.com/python-discord/bot/compare/799d343e24faee34f812ad124e6d3cb43dff69a5..a0913e8c501f1be3c35800d3fb4d31d3cabea34f

**Pull request merged**